chain-signatures/contract/src/errors.rs

```diff
@@ -77,7 +87,7 @@ pub enum MpcContractError {
     #[error("respond fn error: {0}")]
     RespondError(RespondError),
     #[error("vote_* fn error: {0}")]
-    VoteError(VoteError),
+    VoteError(#[from] VoteError),
```

Comment:
Does this help with conflicting `impl near_sdk::FunctionError for VoteError`?

Reply:
This is more for calling into residuals like `err?` so we can forward the error as a `MpcContractError`